tests/modules/text_generation/test_peft_config.py

```diff
@@ -63,3 +70,26 @@ def test_get_peft_config(train_kwargs, dummy_model, request):
     assert peft_config.task_type == dummy_resource.TASK_TYPE
     assert peft_config.prompt_tuning_init == tuning_config.prompt_tuning_init_method
     assert peft_config.prompt_tuning_init_text == tuning_config.prompt_tuning_init_text
+
```

Comment:
q - is there a test case where any of the `if get_config().base_models_dir` block gets tested since that's empty by default in the config?